### DISCUSSION TITLE
Authentication mistake with GitHub and VS Code

### DISCUSSION THREAD

**Discussion by Katyloo** [community/community#39073] [New to GitHub]
Hi Guys, I was setting up my GitHub account to be synced with the VS Code desktop application I just downloaded, but accidentally denied google chrome permission at the authentication web page and now I can't seem to undo my mistake.

I've unknowingly managed to sync my GitHub account with the VS Code app using a one time code from Github, but I'm still being denied permission; for example from the "Working with GitHub in VS Code" web page I'm trying to download the the GitHub Pull Requests and Issues extension, but I keep getting a window popping up saying "The application Google Chrome does not have permission to open......."

I've tried talking to google support, but they're sure it's an issue with the application, would really appreciate any help please!

<img width="254" alt="Screenshot 2022-11-14 at 14 10 41" src="https://user-images.githubusercontent.com/116717109/201670910-d68ebc47-8b60-4fca-ad86-a4b0ea143b83.png">

**Comment by sr229** [comment]
This is not a problem with the Authentication, you have accidentally denied Google Chrome to open URLs, you might want to check through your settings (Cmd+I) and grant the permissions yourself.

**Reply by Imani219** [reply]
Wait I just did the same thing how do I go back and grant it access?
